scripts/tests.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+test_package_with_feature() {
+    local package=$1
+    local features=$2
+
+    /bin/echo -e "\e[0;33m***** Testing '$package' with features '$features' *****\e[0m\n"
+    cargo test --features $features --manifest-path $package/Cargo.toml --no-default-features
+}
+
+cargo fmt --all -- --check
+
+test_package_with_feature primitives std
+
+test_package_with_feature zrml/swaps std
+
+test_package_with_feature zrml/orderbook-v1 std
+
+test_package_with_feature zrml/prediction-markets std
```

Comment:
It's easy to forget to add a line here when a new pallet is created, can we make sure we include all pallets in zrml?

Somethig like
```sh
for package in zrml/*
do
  test_package_with_feature "$package" std
done
```

Reply:
Oh yeah, much better! Thanks